tools/flaky-test-reporter/config.go

```diff
@@ -34,10 +34,28 @@ const (
 )
 
 var (
-	jobConfigs = []JobConfig{
-		{"ci-knative-serving-continuous", "serving", prow.PostsubmitJob}, // CI flow for serving repo
+	// jobConfigs lists all repos and jobs to run within those repos
+	jobConfigs = map[string][]JobConfig{
+		// CI flows for serving repo
+		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-mesh", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+		},
 	}
-	repoIssueMap = map[string]string{
+	// slackChannelsMap lists which slack channel to post results in for each job in repo
+	slackChannelsMap = map[string]map[string][]slackChannel{
```

Comment:
add a `getSlackChannel` helper function here maybe? so caller doesn't need to worry about fallback logic

Reply:
by the way, thank you for moving this logic to a more centralized place

Reply:
Added helper function to slack_notification.go